### ISSUE TITLE
Subscriptions stop working

### ISSUE THREAD

**Issue by npaton** [loro-dev/loro#524]
In JS (tested in browser only, chromium, webkit), subscriptions (subscribeLocalUpdates and subscribe) stop working relatively quickly if the unsubscribe return value is not kept around.

I imagine it has to do with GC, I saw `Avoid auto unsubscribe (due to gc) in js env (#516)`.

In short:

```js
doc.subscribe((evt) => …)
```

Will stop firing after a few events.

```js
const unsub = doc.subscribe((evt) => …)
```

Works fine, if unsub is kept around.

In most cases, you'd want to keep the unsub around, so this is probably not a huge problem, but I got caught by it while testing things, as I didn't care about unsubscribing at that point.

Version tested: 1.0.0-alpha.5.

Great project! 🙏

**Comment by zxch3n** [comment]
I tested it again and confirmed that my failed test case was fixed in #516. Can you clear your test bundler's cache and test it again? It may be using an old version of the WASM binary. For Vite, you can run your dev server by `vite --force` to refresh the cache.

**Comment by zxch3n** [comment]
Fixed in 1.0.0-beta.5
